docs/_Research-Data-Management/03-md.md

```diff
@@ -57,46 +56,61 @@ In microbiology, metadata provides crucial contextual information about biologic
    - **Ethical Approval:** Institutional review board (IRB) approval number
    - **Data Sharing Policy:** Conditions for data sharing, data access restrictions
 
-For more details on the distinction between different types of metadata,  we refer you to the FAIR Cookbook recipe [FAIR and the notion of metadata](https://w3id.org/faircookbook/FCB068) section.
+For more details on the distinction between different types of metadata,  we refer you to the FAIR Cookbook recipe [FAIR and the notion of metadata](https://w3id.org/faircookbook/FCB068).
 
 ## When should you collect your metadata
 
-As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this Knowledge Base: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.
-
-Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will I still understand in a year what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 
+As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, learn how [**Data Management Plans (DMPs)**](./08-dmp.md) could help to plan your experiments and your research data life cycle.
 
-These and other considerations should be thoroughly thought out before the start of your experimental procedures. Some of the metadata can even be collected and documented before starting the experiments if you already know how to collect your samples, process, sequence them (if sequencing is a part of the analysis), and analyze them. 
+Metadata collection should be planned, but at the same time, it can be overwhelming. <br>
```

Comment:
```suggestion
Metadata collection should be planned, but at the same time, it can be overwhelming.


```

You can also used two blank lines to create new paragraph blocks without the ~~ugly~~ `<br>` tag